frontends/web/src/routes/buy/info.tsx

```diff
@@ -74,19 +73,33 @@ class BuyInfo extends Component<Props, State> {
     )
       .then(results => results.filter(result => result))
     // @ts-ignore
```

Comment:
since you are here, could you try to get rid of this  `// @ts-ignore`?

Reply:
Yeah. Gotta explicitly cast as `IAccount[]` though because otherwise the promises return `false | IAccount[].